python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -150,33 +151,51 @@ def build(self, measure_inputs):
                             )
 
                         else:
-                            if "InstantiationError" in str(res.error):
-                                msg = str(res.error)
+                            tb, exception = res.error
+                            if "InstantiationError" in str(exception):
+                                msg = str(exception)
                                 try:
                                     msg = msg.split("\n")[-2].split(": ")[1]
                                 except Exception:  # pylint: disable=broad-except
                                     pass
                                 res = MeasureResult(
-                                    (InstantiationError(msg),),
+                                    (
+                                        tb,
+                                        InstantiationError(msg),
+                                    ),
```

Comment:
IIUC, from this structure and https://github.com/apache/tvm/blob/main/python/tvm/autotvm/tuner/tuner.py#L144, `InstantiationError(msg)` is now only in the tuning log instead of logging out (in the debug mode). Is this intentional? Or maybe we just concat tb and exception to have something like `InstantiationError(tb + "\n" + exception)`?